features/src/main/feature/camel-features.xml

```diff
@@ -285,11 +285,18 @@
 
     <feature name='camel-activemq' version='${project.version}' start-level='50'>
         <feature version='${camel.osgi.version.range}'>camel-jms</feature>
+        <bundle dependency='true'>mvn:org.apache.activemq/activemq-client-jakarta/${activemq-version}</bundle>
```

Comment:
NB: when upgrading to ActiveMQ 6.x, we will be able to use `activemq-client` (`activemq-client-jakarta` is just for transition on 5.18.x).